XiEditor/EditViewController.swift

```diff
@@ -442,7 +442,11 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
     @IBAction func debugRewrap(_ sender: AnyObject) {
         document.sendRpcAsync("debug_rewrap", params: [])
     }
-    
+
+    @IBAction func debugWrapWidth(_ sender: AnyObject) {
+        document.sendRpcAsync("debug_wrap_width", params: [])
```

Comment:
We should start thinking about what the actual protocol changes in support of this should look like.